home-assistant-plugin/custom_components/xknx/__init__.py

```diff
@@ -148,11 +148,13 @@ async def async_setup(hass, config):
                 create_knx_device(
                     hass, platform, hass.data[DATA_XKNX].xknx, device_config
                 )
-            hass.async_create_task(
-                discovery.async_load_platform(
-                    hass, platform.value, DOMAIN, {}, config
-                )
+
```

Comment:
```suggestion
    # load platforms when creation of all devices is finished (weather sensor exposures would not be loaded)
```
add a comment to remember why we do the same loop twice